extensions/agroal/deployment/pom.xml

```diff
@@ -31,6 +31,13 @@
             <artifactId>quarkus-narayana-jta-deployment</artifactId>
         </dependency>
 
+        <!-- Add the health extension as optional as we will produce the health check only if it's included -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-health</artifactId>
```

Comment:
is `-deployment` missing from the artifactId?

Reply:
Agroal only needs the runtime dependency to be able to compile the health check. Moreover the dependency is optional to avoid adding it in case agroal is used without smallrye-health.

In fact we can witch to io.smallrye:smallrye-health dependency instead of Quarkus one if we want as there is no dependency to the extention ...